### ISSUE TITLE
Use UI dialogs from `brainglobe-utils`

### ISSUE THREAD

**Issue by adamltyson** [brainglobe/brainglobe.github.io#224]
Rather than using Qt directly, using [these](https://github.com/brainglobe/brainglobe-utils/blob/main/brainglobe_utils/qtpy/dialog.py) would make the code very slightly easier to maintain. By default it will also make the dialogs match napari.

**Comment by alessandrofelder** [comment]
Is this in the right repo? Should this be in `brainrender-napari` or other repo with napari plugins? If not, could you clarify this issue, please?

**Comment by adamltyson** [comment]
Yes, sorry it should be in `brainrender-napari`. I accidentality raised it here and forgot to transfer it. Will do it now.
